lib/view_model/active_record/nested_controller_base.rb

```diff
@@ -8,14 +8,16 @@ def show_association(scope: nil, serialize_context: new_serialize_context)
     associated_views = nil
     pre_rendered = owner_viewmodel.transaction do
       owner_view = owner_viewmodel.find(owner_viewmodel_id, eager_include: false, serialize_context: serialize_context)
-      serialize_context.visible!(owner_view)
+      serialize_context.run_callback(ViewModel::Callbacks::Hook::BeforeVisit, owner_view)
       # Association manipulation methods construct child contexts internally
       associated_views = owner_view.load_associated(association_name, scope: scope, serialize_context: serialize_context)
 
       associated_views = yield(associated_views) if block_given?
 
       child_context = serialize_context.for_child(owner_viewmodel, association_name: association_name)
-      prerender_viewmodel(associated_views, serialize_context: child_context)
+      json = prerender_viewmodel(associated_views, serialize_context: child_context)
```

Comment:
I had to go look at the implementation to see if this was actually json. Everywhere else leaves it opaque. Consider `.tap` or assigning to `pre_rendered` explicitly.

Reply:
done